<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>831</width>
    <height>424</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(20, 114, 255);</string>
  </property>
  <widget class="QPushButton" name="btnback">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>380</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(232, 232, 232);</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>781</width>
     <height>271</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(232, 232, 232);</string>
   </property>
   <property name="rowCount">
    <number>0</number>
   </property>
   <property name="columnCount">
    <number>7</number>
   </property>
   <column>
    <property name="text">
     <string>id</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>nip</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>nama</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>dept</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>tanggal</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>jam masuk</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>jam keluar</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>30</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Data Absensi Karyawan</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
